vendor/sigs.k8s.io/cloud-provider-azure/pkg/provider/azure_backoff.go

```diff
@@ -384,7 +384,7 @@ func (az *Cloud) DeletePublicIP(service *v1.Service, pipResourceGroup string, pi
 }
 
 // DeleteLB invokes az.LoadBalancerClient.Delete with exponential backoff retry
-func (az *Cloud) DeleteLB(service *v1.Service, lbName string) error {
```

Comment:
You have updates to the `vendor` directory, but I don't see any corresponding updates to `go.mod`. Are you missing this file in the PR?

Reply:
I must have forgotten to add go.mod. When I updated the vendors, it changed almost all of the packages so I was trying to just include the ones relevant. I'll fix that

Reply:
vendor change was already done in previous PR, just rebase to current master branch